pkg/deploy/lattice/target_group_synthesizer_test.go

```diff
@@ -129,17 +129,12 @@ func Test_SynthesizeTriggeredServiceExport(t *testing.T) {
 				TargetGroupID:  "4567",
 			}
 
-			if tg.Spec.IsDeleted {
+			if !tg.Spec.IsDeleted {
 				if tt.tgManagerError {
-					mockTGManager.EXPECT().Delete(ctx, tg).Return(errors.New("ERROR"))
-				} else {
-					mockTGManager.EXPECT().Delete(ctx, tg).Return(nil)
-				}
-			} else {
-				if tt.tgManagerError {
-					mockTGManager.EXPECT().Create(ctx, tg).Return(tgStatus, errors.New("ERROR"))
+					mockTGManager.EXPECT().Get(ctx, tg).Return(tgStatus, errors.New("ERROR"))
+
 				} else {
-					mockTGManager.EXPECT().Create(ctx, tg).Return(tgStatus, nil)
+					mockTGManager.EXPECT().Get(ctx, tg).Return(tgStatus, nil)
 				}
```

Comment:
When create or delete serviceExport, controller should not make TGManager call Create() or Delete()
When create serviceExport, controller should  make TGManager call Get()
Only when create or delete  `Kind: Service`, controller should make TGManager call Create() or Delete()